android/src/main/kotlin/com/wromance/plugins/audioadmob/AudiomobHostApi.kt

```
@@ -0,0 +1,170 @@
+
+class AudiomobHostApiImplementation(
+    private val audiomobPlugin: AudiomobPlugin
+) : AudiomobHostApi {
+    override fun initialize(
+        apiKey: String,
+        bundleId: String,
+        backgroundModeEnabled: Boolean,
+    ) {
+        try {
+            audiomobPlugin.initialise(apiKey, bundleId, backgroundModeEnabled)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)